BRANDING.MD

```diff
@@ -75,17 +75,11 @@ Suggested templates:
 * Don't use the Knative name or logo to indicate that your design or
   implementation is compatible with Knative without prior written permission. 
 
-The Knative Trademark Committee understands the community desire for a
-conformance standard, and we do plan to implement one at a future date. We
-also recognize that conformance is a complicated issue with many variables,
-and it is still early days for the Knative project. For comparison, the
-Kubernetes project did not implement the [Certified Kubernetes Conformance
-Program](https://github.com/cncf/k8s-conformance) until version 1.7. Our goal
-is to have a Conformance Working Group created soon.
```

Comment:
Why remove this line? _Our goal is to have a Conformance Working Group created soon._

I believe is a good goal to have a team focusing on conformance with leadership and cadence, also the link added [Knative Conformance Certification](https://github.com/knative/specs/blob/main/docs/conformance_procedure.md) it has a lot of TODO and blanks, I hope the working group move this effort forward.

Reply:
Those lines were from 2020 and we now have a conformance process. I don't object to leaving in that, though, but it is a little odd that we hope to have one soon was written 2 years ago.